pkg/daemon/ceph/client/pool.go

```diff
@@ -239,12 +239,12 @@ func CreateECPoolForApp(context *clusterd.Context, clusterName string, newPool C
 		}
 	}
 
-	if !newPool.NotEnableAppPool {
-		err = givePoolAppTag(context, clusterName, newPool.Name, appName)
-		if err != nil {
-			return err
-		}
-	}
+	// if !newPool.NotEnableAppPool {
+	// 	err = givePoolAppTag(context, clusterName, newPool.Name, appName)
+	// 	if err != nil {
+	// 		return err
+	// 	}
+	// }
```

Comment:
drop the commented code

Reply:
Still in WIP, just wanted you guys to review it first.

Reply:
@rajatsingh25aug A PR should be matured enough to be reviewed. The commented lines of code are not acceptable in a PR. I'd suggest you to close this PR and submit it again when your work and testing are finished.

Reply:
Keep working on this PR and add a better commit message.